custom/src/main/java/com/splunk/opentelemetry/RuleBasedSamplerProvider.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.contrib.samplers.RuleBasedRoutingSampler;
+import io.opentelemetry.contrib.samplers.RuleBasedRoutingSamplerBuilder;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import io.opentelemetry.sdk.autoconfigure.spi.traces.ConfigurableSamplerProvider;
+import io.opentelemetry.sdk.trace.samplers.Sampler;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import java.util.ArrayList;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@AutoService(ConfigurableSamplerProvider.class)
+public class RuleBasedSamplerProvider implements ConfigurableSamplerProvider {
+  private static final Logger log = LoggerFactory.getLogger(RuleBasedSamplerProvider.class);
+  public static final String SAMPLER_ARG = "otel.traces.sampler.arg";
+
+  @Override
+  public Sampler createSampler(ConfigProperties config) {
+    String configString = config.getString(SAMPLER_ARG);
+    Config samplerConfiguration = new Config(configString);
+
+    log.info("Received following rules: {}", samplerConfiguration);
+
+    RuleBasedRoutingSamplerBuilder builder =
+        RuleBasedRoutingSampler.builder(SpanKind.SERVER, samplerConfiguration.fallback);
```

Comment:
The docs don't mention anything about it being a `SERVER` sampler - should they?